packages/kit-docs/src/lib/components/layout/KitDocsLayout.svelte

```diff
@@ -237,9 +239,11 @@
       <slot name="main-top" />
 
       {#if $kitDocs.meta}
-        <article class="markdown prose dark:prose-invert z-10 max-w-[var(--kd-article-max-width)]">
+        <article
+          class="markdown kd-prose dark:kd-prose-invert kd-z-10 kd-max-w-[var(--kd-article-max-width)]"
```

Comment:
Known issue is here. `dark:kd-prose-invert` doesn't work for some unknown reason.